webapp/scripts/query.js

```diff
@@ -1,17 +1,20 @@
 const fs = require('fs')
-const { checkWritableDir, getNumericColumns, getTopology, gpkgOut, initMapset, grass, mapsetExists, mergePDFs, psToPDF, textToPS, remove, getUnivar, getUnivarBounds } = require('./functions')
-const { module_2: messages } = require('./messages.json')
+const { checkWritableDir, getNumericColumns, getTopology, gpkgOut, initMapset, grass, mapsetExists, mergePDFs, psToPDF, textToPS, remove, getUnivar, getUnivarBounds, getAllFile, listUserVector, addVector } = require('./functions')
+const { query: messages } = require('./messages.json')
 
 const GEOSERVER = `${process.env.GEOSERVER_DATA_DIR}/data`
+const CONTAINER_GEOSERVER = '/usr/share/geoserver/data_dir/data'
```

Comment:
What's the idea here? It shouldn't be anything else than `${process.env.GEOSERVER_DATA_DIR}/data`